components/02-content/accordion/light/accordion-light.js

```diff
@@ -0,0 +1,37 @@
+// Notes:
+// ------
+// * Each change to this file needs a restart of the fractal instance
+// * Exported methods are merged as helpers in fractal.js
+
+module.exports = function () {
+  return {
+    accordionLight(options) {
+      const classNames = ['dso-accordion-light', this.modifiers].filter(c => c).join(' ');
+
+      return `
+        <div class="${classNames}">
+          ${options.fn(this)}
+        </div>
+      `.trim();
+    },
+
+    accordionLightSection(title, options) {
+      const self = this;
+      const header = 'h3';
```

Comment:
Een const kan toch nooit wijzigen? Waarom dan niet op regel 23 keihard een h3 zetten? @tfrijsewijk wat vind jij?

Reply:
Nee, mijn fout. Die const geldt per each en kan in die each nooit meer wijzigen. Wat echter niet klopt is dat hij gevuld wordt met h3. De default doet dit: const header = options.hash.header || 'div';

Reply:
Angepast zodat div, h1 t/m 4 getoond worden in de markup